app/controllers/attachments_controller.rb

```diff
@@ -3,14 +3,16 @@
 #
 class AttachmentsController < ApplicationController
   include FragmentCachable
+  include InfoRequestHelper
 
   around_action :cache_attachments
 
   before_action :find_info_request, :find_incoming_message, :find_attachment
-  before_action :generate_attachment_url
   before_action :authenticate_attachment
   before_action :authenticate_attachment_as_html, only: :show_as_html
 
+  helper_method :info_request, :incoming_message, :attachment
```

Comment:
Looks wrong – we don't have `:info_request` and `:incoming_message` helpers

Reply:
I don't think we want the `:attachment` helper method here either, since we're still using `@attachment`.

Reply:
Oops thats a bad rebase